webapp/src/browserworkspace.ts

```diff
@@ -1,28 +1,35 @@
 import * as db from "./db";
 
-let headers: db.Table;
-let texts: db.Table;
+let headerDb: db.Table;
+let textDb: db.Table;
 
 type Header = pxt.workspace.Header;
 type ScriptText = pxt.workspace.ScriptText;
 type WorkspaceProvider = pxt.workspace.WorkspaceProvider;
 
+type TextDbEntry = {
+    files?: ScriptText,
+    // These are required by PouchDB/CouchDB
+    id: string,
+    _rev: any // This must be set to the return value of the last PouchDB/CouchDB
```

Comment:
`_rev` is a reserved name in PouchDB. Is this intended to map to the builtin field?

Reply:
Yes, this is reserving the "_rev" field for use by PouchDB. But also we have to access this and reason about it in our code so it makes sense to type it IMO